packages/router/src/grpc/view-protocol-server/status.ts

```diff
@@ -9,7 +9,7 @@ export const status: Impl['status'] = async (_, ctx) => {
   if (!fullSyncHeight) throw new Error('Last block synced not in db');
 
   return {
-    catchingUp: fullSyncHeight === latestBlockHeight,
+    catchingUp: fullSyncHeight !== latestBlockHeight,
```

Comment:
Ah good catch. It's an edge case (maybe an error somewhere else if so), but should we do:

```
catchingUp: fullSyncHeight < latestBlockHeight,
```

Reply:
https://github.com/penumbra-zone/penumbra/blob/d3bbe4c920dd03981ed883d347ed579dae6af363/crates/view/src/service.rs#L326-L334
rust view service also returns `catchingUp=false` if the difference is only 1 block

Reply:
ah, nice find